ui/apps/everest/src/components/cluster-form/advanced-configuration/advanced-configuration-schema.ts

```diff
@@ -0,0 +1,51 @@
+// everest
+// Copyright (C) 2023 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { z } from 'zod';
+import { AdvancedConfigurationFields } from './advanced-configuration.types';
+import { IP_REGEX } from 'consts';
+import { Messages } from './messages';
+
+export const advancedConfigurationsSchema = z
+  .object({
+    [AdvancedConfigurationFields.externalAccess]: z.boolean(),
+    // internetFacing: z.boolean(),
```

Comment:
do we still need this comment?

Reply:
good question!i will ask will we implement this in nearest future or not

Reply:
@dianabirs it will be removed (https://github.com/percona/everest/pull/771/commits/94c3b408c84ca91292966d14eaa061e9d44d2a48)